metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -462,7 +462,7 @@ void getCreatePublicationRequestReturnsOptionalEmptyWhenExpectedInputIsStringBut
     }
 
     @ParameterizedTest(name = "Emtpy value and error logged when invalid value is passed to {0}")
-    @ValueSource(strings = {"citation_isbn", "citation_issn"})
+    @ValueSource(strings = {"citation_issn"})
     void getCreatePublicationRequestReturnsOptionalEmptyWhenExpectedInputIsInvalidIsxn(String property)
```

Comment:
No need to be parameterized test longer? 
Do we need to remove isbn from the test?  Rename Isxn to Issn in the test name

Reply:
Is isbn errors something we now can receive when you changed nva-datamodel?

Reply:
Changed test from parameterised test.